<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="verticalTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Fichier/Dessin</string>
     </attribute>
     <widget class="QTabWidget" name="dessin">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>9</y>
        <width>591</width>
        <height>461</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="gridTabWidget_2Page1">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="testmoveButton">
          <property name="text">
           <string>Move 1 X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Dessin</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>479</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Librairie</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>481</width>
          <height>200</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="0">
          <widget class="QTreeWidget" name="browserWidget">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Driver/Code machine</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>463</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Driver file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Driver state</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="gcodesendText"/>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="getstatutButton">
         <property name="text">
          <string>Statut driver?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="deviceMenu">
         <item>
          <property name="text">
           <string>/dev/minicut_virtmnc</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/parrot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/hidraw0</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="initButton">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="dri_statelabel">
         <property name="text">
          <string>inconnu</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="dev_filelabel">
         <property name="text">
          <string>inconnu</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>./GRBL.ngc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gcodesendButton">
           <property name="text">
            <string>Envoyer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Importer GRBL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="drawtocodeButton">
           <property name="text">
            <string>Générer le code!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalTabWidgetPage1">
     <attribute name="title">
      <string>Pilotage manuel</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="moinschaudButton">
          <property name="text">
           <string>MOINS chaud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="tempbare">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pluschaudButton">
          <property name="text">
           <string>PLUS chaud</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="bgButton">
          <property name="text">
           <string>GAUCHE/BAS</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="hButton">
          <property name="text">
           <string>HAUT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="bButton">
          <property name="text">
           <string>BAS</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="hgButton">
          <property name="text">
           <string>GAUCHE/HAUT</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="hdButton">
          <property name="text">
           <string>DROITE/HAUT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="bdButton">
          <property name="text">
           <string>DROITE/BAS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="gButton">
          <property name="text">
           <string>GAUCHE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="dButton">
          <property name="text">
           <string>DROITE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="runstopbutton">
            <property name="text">
             <string>Run/Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="movewaylabel">
            <property name="text">
             <string>ordre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuM">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuStatut">
    <property name="title">
     <string>Machine</string>
    </property>
    <addaction name="actionV_rifier_mahine"/>
    <addaction name="actionNouvelle"/>
    <addaction name="actionConnecter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionManuel"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuScreen_size">
     <property name="title">
      <string>Screen size</string>
     </property>
     <addaction name="actionLittle_height_1024px"/>
     <addaction name="actionNormal_15_fullscreen_test"/>
     <addaction name="actionAutodetect"/>
    </widget>
    <addaction name="actionLibrary_path"/>
    <addaction name="menuScreen_size"/>
    <addaction name="actionToggle_Library"/>
   </widget>
   <addaction name="menuM"/>
   <addaction name="menuStatut"/>
   <addaction name="menuAide"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionV_rifier_mahine">
   <property name="text">
    <string>Vérifier machine</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionNouvelle">
   <property name="text">
    <string>Nouvelle...</string>
   </property>
  </action>
  <action name="actionConnecter">
   <property name="text">
    <string>Connecter...</string>
   </property>
  </action>
  <action name="actionLibrary_path">
   <property name="text">
    <string>Library path...</string>
   </property>
  </action>
  <action name="actionLittle_height_1024px">
   <property name="text">
    <string>Little(height&lt;960px)</string>
   </property>
  </action>
  <action name="actionNormal_15_fullscreen_test">
   <property name="text">
    <string>normal(15&quot; fullscreen test)</string>
   </property>
  </action>
  <action name="actionAutodetect">
   <property name="text">
    <string>Autodetect</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionToggle_Library">
   <property name="text">
    <string>Toggle Library</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
